app/helpers/errorSummaryHandlers.js

```diff
@@ -185,6 +185,8 @@ const checkErrors = (payload, currentQuestion, h, request) => {
 }
 
 module.exports = {
+  validateAnswerField,
+  checkInputError,
```

Comment:
Functions should not be exported. 

Either: 
1. Use an npm package (such as **rewire**) to test these two specific functions (replaces need to export them)
2. Create tests for **checkErrors** and **customiseErrorText** functions (writing tests for these two functions will also test **validateAnswerField** and **checkInputError**)